mlrun/feature_store/api.py

```diff
@@ -476,6 +476,10 @@ def _ingest_with_spark(
         targets = [get_target_driver(target, featureset) for target in targets]
 
     for target in targets or []:
+        if target.path and urlparse(target.path).scheme == "":
```

Comment:
This is post spark session creation, I guess that the whole block has to be inside try-finally with _post_ingestion (not only your code, but especially this exception).